### DISCUSSION TITLE
New Destination OceanBase

### DISCUSSION THREAD

**Discussion by yuanoOo** [airbytehq/airbyte#39391] [New Connector Request]
[OceanBase](https://en.oceanbase.com/) is an Ultra-Fast & Cost-Effective Distributed SQL Database. 
Unlimited scalable distributed database for data-intensive transactional and real-time operational analytics workloads, with ultra-fast performance that has once achieved world records in the TPC-C benchmark test. OceanBase has served over 1000 customers across the globe and has been supporting all mission critical systems in Alipay.

We want to be able to sync data to OceanBase via Airbyte. <br/> So, I will try to develop this new Airbyte connector. Does anyone have any suggestions?

**Comment by yuanoOo** [comment]
I developed the connector according to the documentation. https://docs.airbyte.com/connector-development/tutorials/building-a-java-destination
In the first step: Create the destination using the template:
But I cannot find the Java Destination after run `./generate.sh`
![20240613111954](https://github.com/airbytehq/airbyte/assets/24812548/c8660970-9b45-443a-911d-b9d8f08c035a)


@marcosmarxm Take a look please.

**Reply by marcosmarxm** [reply]
The project is not accepting new Python destinations @yuanoOo

**Reply by yuanoOo** [reply]
Hi, what about the new Java destination? In this discussion, I'm going to contribute new Java destinations to the project.
Does this project accept new Java destinations? If not, which project should I submit to? Looking forward to your reply and hoping for a clear answer.  @marcosmarxm

[a comment by an automated account is not shown]
